src/main/java/bio/terra/controller/GlobalExceptionHandler.java

```diff
@@ -54,7 +65,7 @@ public ErrorModel catchallHandler(Exception ex) {
         return buildErrorModel(ex);
     }
 
-    private ErrorModel buildErrorModel(Exception ex) {
+    private ErrorModel buildErrorModel(Throwable ex) {
```

Comment:
since you changed this to Throwable, should we also change our catchallHandler to handle Throwable?

Reply:
Sure. I changed it to Throwable, because I am pulling out the "cause" from the exception to unwrap the nesting.